docs/file_explorer.md

```diff
@@ -1,68 +1,116 @@
 
 File Storage
 ------------
 By default, Remix IDE stores files in **IndexedDB**.  
 
-Coding in Remix IDE Online is different from writing in a Google doc. Yes, both are written in a browser but a Google doc saves your work to Google’s servers, and Remix—out of the box—only saves your code to your browser’s storage. So tread carefully, browser storage is not permanent!
+Coding in Remix IDE Online is different from writing in a Google doc. 
+- A Google doc saves your work to your account on Google’s servers.
+- Remix has no user accounts. By default, files are ONLY saved locally in the browser’s storage. So tread carefully, browser storage is not permanent!
+
+It is very important to have a file backup & file saving strategy. 
+
+**Check the following techniques for:**
+- [saving to your hard drive](#file-storage-on-your-hard-drive)
+- using [remote Git repos](#connecting-remix-to-remote-git-repos)
+- [backing up workspaces](#backup).
 
-**Important Note:** Clearing the browser storage & IndexedDB will **permanently delete** all the files stored there. 
+**Important Note:** Clearing the browser storage & IndexedDB will **permanently delete** all the files stored there. It is prudent to backup your workspaces before deleting them...just in case. 
 
-## File Storage outside of the browser and Version Control
+### File Storage on your hard drive
+#### Remixd
+For storing files on your computer's hard drive when using Remix Online IDE, use **[Remixd](remixd.html)** 
+- Remixd is an NPM module that runs on your computer. It allows you to share a specified folder on your computer with Remix IDE. 
 
-If you want to use browser storage, but also to save a git repo on IPFS, use the **DGIT** plugin.  
+#### Remix Desktop
+Remix Desktop is a version of Remix IDE in an Electron app. Note that when using Remix Desktop, you cannot use a browser wallet like MetaMask, because Remix Desktop does not run in a browser.  To connect to public networks, you need to use service like Infura and then use the WalletConnect plugin to approve transactions on your mobile device.
 
-If you want to store files on your computer's filesystem, use **[Remixd](remixd.html)** or use the **[desktop version of Remix IDE](https://github.com/ethereum/remix-desktop/releases/)**. Remixd enables you to have access to a selected folder on your hard drive. Remix Desktop is a version of Remix IDE in an Electron app. 
+### Connecting Remix to remote Git repos
+**If you are not using Remixd, it is highly recommended that you save to a remote repo.** 
+(Browsers do crash causing localstorage & indexedDB to be corrupted)
+
+Remix IDE can be connected to remote Git repos hosted in GitHub (or similar service) or in IPFS.  
+Most of the Git operations are done in the **DGIT** pluin. (DGIT stands for Decentralized GIT).
+
+Branch management is also available at the bottom of the File Explorer when the Workspace is Git initialized.
+(see this page for the Git workflow).
```

Comment:
Are you considering to add a link on `this page`

Reply:
Updated - but I will be adding a separate page about Git in Remix.